File: app/src/main/java/com/dgd/earthquakes/data/network/INetworkManager.java
package com.dgd.earthquakes.data.network;

import com.dgd.earthquakes.data.network.callback.IQuakesCallbackListener;

import java.util.Date;

/**
 * Created by Max
 * on 23-Jun-17.
 */

public interface INetworkManager {
    void getEarthquakes(Date start, Date end, IQuakesCallbackListener listener);

    void checkNewEarthquakes(long lastUpdateDate, IQuakesCallbackListener listener);
}

File: app/src/main/java/com/dgd/earthquakes/data/network/INetworkManager.kt
package com.dgd.earthquakes.data.network

import com.dgd.earthquakes.data.network.callback.IQuakesCallbackListener
import java.util.*

/**
 * Created by Max
 * on 23-Jun-17.
 */

interface INetworkManager {
    fun getEarthquakes(start: Date, end: Date, listener: IQuakesCallbackListener)

    fun checkNewEarthquakes(listener: IQuakesCallbackListener)
}
